daisy_workflows/build-publish/rhui/install_cds.sh

```diff
@@ -89,6 +89,14 @@ cat $tempdir/rhui-tools.conf > /etc/rhui/rhui-tools.conf
 # Remove enrollment cert and repos from final image.
 subscription-manager remove --all
 
+# Install health checks
+install -g root -o root -D -t /opt/google-rhui-infra $tempdir/health_check.py
```

Comment:
nit but this is overkill - the script is already running as root. i also had to look up all these args :laughing:

Reply:
Updated to remove the user and group